src/js/modules/Support.js

```diff
@@ -68,44 +84,127 @@ export class Support {
 
           const $button = $(this);
           const data = $button.data();
-
+          // function to update the support section copy depending on the type of donation
+          function updateSupportSection(data, $form) {
```

Comment:
this function should be a general method on the class for this functionality otherwise it will be being redeclared multiple times.